Simplified/Book/UI/NYPLBookButtonsView.m

```diff
@@ -303,9 +303,21 @@ - (void)updateButtons
       [self.book.defaultAcquisition.availability
        matchUnavailable:nil
        limited:^(NYPLOPDSAcquisitionAvailabilityLimited *const _Nonnull limited) {
+        
+        NSDate *untilDate = self.book.defaultAcquisition.availability.until;
+        NSDate *sinceDate = self.book.defaultAcquisition.availability.since;
+        
+        BOOL hasIncorrectDates = FALSE;
+        
+        if (untilDate && sinceDate) {
+          hasIncorrectDates = (untilDate.timeIntervalSinceReferenceDate - sinceDate.timeIntervalSinceReferenceDate == 3600);
```

Comment:
all this logic should at the very least go inside a #ifdef AXIS since it's only affecting Axis. But honestly I don't think it's the job of the app to patch these issues. I'll elaborate in the ticket.

Reply:
Agreed.